src/vscode-bicep/jest.config.unit.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+module.exports = {
+  roots: ["<rootDir>/out/test/unit"],
+  testMatch: ["<rootDir>/out/test/unit/**/*.test.js"],
+  verbose: true,
```

Comment:
I didn't turn on code coverage because we only have unit tests for one file and the coverage is very low, but if we want we can simply add `collectCoverage: true` to this file to get coverage reports.

Reply:
That's a good reason to turn on code coverage.

---
In reply to: [508194276](https://github.com/Azure/bicep/pull/686#discussion_r508194276) [](ancestors = 508194276)

Reply:
I'm sold. I turned it on. We are at 25.98% now. I'll send follow-up PRs to bump the number.